src/packages/name-service/utils.ts

```diff
@@ -1,12 +1,14 @@
 import { AssertionError, deepStrictEqual } from "assert";
 import { getLogger } from "../..";
+import config from "../../config";
 import { ErrorHelper, PackageErrorCode } from "../error";
 import { cachedFunctionCall, httpJSONRequest } from "../utils";
 
 const log = getLogger(__filename);
 
-export const fetchNameDetails = async (blockstackId: string, bnsNodes: string[]): Promise<object|undefined> => {
-    const nodeResponses = bnsNodes.map((baseUrl) => bnsResolveName(baseUrl, blockstackId));
+export const fetchNameDetails = async (blockstackId: string, bnsNodes?: string[]): Promise<object|undefined> => {
+    bnsNodes = bnsNodes || config.BLOCKSTACK.BNS_NODES;
```

Comment:
Have this be a mandatory param, utils shouldn't have anything to do with config.